pkg/get/tools.go

```diff
@@ -463,21 +463,23 @@ https://storage.googleapis.com/kubernetes-release/release/{{.Version}}/bin/{{$os
 {{- end -}}
 https://github.com/inlets/inletsctl/releases/download/{{.Version}}/{{$fileName}}`,
 			BinaryTemplate: `{{ if HasPrefix .OS "ming" -}}
-{{.Name}}.exe
+{{.Name}}
```

Comment:
If these needed to change, why didn't the unit tests also get changed?

If there are none, could you add them please?

Reply:
The URLs expected by the unit tests did not change.

What did change is the expected name of the executable, and there are no unit tests for that anywhere, AFAICT, right?